src/demo/app.ts

```diff
@@ -79,30 +72,26 @@ fin.desktop.main(async () => {
             //     ${notifications.value.length} notifications received from the
             //     Notification Center!
             // `
-            logit(`${notifications.value.length} notifications received from the Notification Center`);
+            logit(`${notifications.length} notifications received from the Notification Center`);
         });
     });
 
-    ofnotes.addEventListener('click', (payload: NotificationEvent, sender: ProviderIdentity) => {
+    ofnotes.addEventListener<NotificationClickedEvent>('notification-clicked', (event: NotificationClickedEvent) => {
```

Comment:
Is `<NotificationClickedEvent>` needed here? Below you use addEventListener with out the type.